learning/katas/go/Introduction/Hello Beam/Hello Beam/pkg/task/task.go

```diff
@@ -0,0 +1,24 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package task
+
+import (
+	"github.com/apache/beam/sdks/go/pkg/beam"
+)
+
+func HelloBeam(s beam.Scope) beam.PCollection {
```

Comment:
I might have confused you on this.
Can we have this in one file with the main() function?
If we can't do that due to the function renaming to "main()", let's revert back to the previous naming so that we can have it in just one file.

Reply:
May we consider keeping this?  When I consolidated everything into the same file, I received the error `flag redefined: runner`.  IntelliJ Edu constrains us to having tests in the test folder in order to achieve lesson task validation, so we can't do any task validation with tests in any other folder.  Also, while there doesn't seem to be a consensus around go project structure in the community, this repo seems to follows the `./{cmd,pkg}/*` structure.

Reply:
Noted that Damon. I'm not that familiar with Go. Just did some reading and I guess this is good for now. Thanks for clarifying.

Also FYI, you may want to consider using GoLand instead of IntelliJ for developing this.

Reply:
Thank you, Henry.  I test on both GoLand and IntelliJ in case individuals are coming from either.  Thank you for embracing my suggestion :).